docs/pages/server-access/rbac.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Server Access Controls
+description: Role-based access control (RBAC) for Teleport Server Access.
+---
+
+# Server Access Role-Based Access Control
+
+Role-based access control (or RBAC, for short) allows administrators to set up
+granular access policies for servers connected to Teleport.
```

Comment:
Can we expand "granular access policies" a bit? Otherwise, this repeats the technical term "role-based access control". For example: "...allows administrators to define rules for which members of an organization can perform operations on servers connected to Teleport."